static/katex.less

```diff
@@ -513,6 +513,15 @@
             // it left. We center it by shifting it half way right again.
             left: 0.326em;
         }
+
+        .accent-body.accent-text > span {
+            position: relative;
+            left: 0.262em;  // width of space, apparently
```

Comment:
The width, at least for KaTeX_Main-Regular, is 0.250em.  Thankfully it appears that this value is the same, at least for the fonts I checked which were KaTeX_Main-Bold and KaTeX_SansSerif-Regular.